configs/default/GEPR-GEPRCG4AIO.config

```diff
@@ -0,0 +1,102 @@
+# Betaflight / STM32G47X (SG47) 4.5.0 Feb 21 2023 / 01:24:44 (d98740b) MSP API: 1.45
+
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25Q128FV
+#define USE_FLASH_W25N01G
```

Comment:
Using 3 different flash chips ???

Reply:
sorry, only added to correct one, and didnt remove the other two. updated :)